src/main/java/com/es/phoneshop/web/ProductDemodataServletContextListener.java

```diff
@@ -0,0 +1,34 @@
+package com.es.phoneshop.web;
```

Comment:
ProductDemodataServletContextListener -> ProductDemoDataServletContextListener

Reply:
what did you mean?